cpp/src/arrow/ipc/read_write_test.cc

```diff
@@ -1289,6 +1338,10 @@ INSTANTIATE_TEST_SUITE_P(StreamDecoderSmallChunksRoundTripTests,
 INSTANTIATE_TEST_SUITE_P(StreamDecoderLargeChunksRoundTripTests,
                          TestStreamDecoderLargeChunks, BATCH_CASES());
 
+// TODO(kiszk): Enable this test since it is not successful now
```

Comment:
Now, I intetionally comment out this test since this type of communication is not supported.